controls/radrichtextbox/import-export/Html/htmlformatprovider.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Using HtmlFormatProvider
+page_title: Using HtmlFormatProvider
+description: Using HtmlFormatProvider
+slug: radrichtextbox-import-export-html-htmlformatprovider
+tags: using-htmlFormatProvider
+published: True
+position: 0
+---
+
+# Using HtmlFormatProvider
+
+__HtmlFormatProvider__ makes it easy to import and export __RadDocument__ to/from HTML format, preserving as much as possible of the document structure and formatting.
+
+To use __HtmlFormatProvider__, you should add references to the assembly listed below:
+
+* __Telerik.Windows.Documents.FormatProviders.Html__
+
+## Import
+
+In order to import an HTML document you can use the overloads of the __HtmlFormatProvider.Import()__ method.
+
+__Example 1__ shows how to use HtmlFormatProvider to import an HTML document from a file.
+
+#### __[C#] Example 1: Import HTML file__
+
+{{region cs-radrichtextbox-import-export-html-htmlformatprovider_0}}
+    using (Stream input = File.Create(@"Sample.html"))
+    {
+        HtmlFormatProvider provider = new HtmlFormatProvider();
+        RadDocument document = provider.Import(input);
+    }
+{{endregion}}
+
+__Example 2__ shows how you can import an HTML string.
+ 
+#### __[C#] Example 2: Import HTML string__
+
+{{region cs-radrichtextbox-import-export-html-htmlformatprovider_1}}
+    string html = "<p>hello world!</p>";
+    HtmlFormatProvider provider = new HtmlFormatProvider();
+    RadDocument document = provider.Import(html);
+{{endregion}}
+
+The resulting __RadDocument__ can be used like any code-generated document.
+        
+## Export
+
+In order to export a document to HTML you can use the overloads of the __HtmlFormatProvider.Export()__ method.
```

Comment:
Let's say something different than the sentence in the above section. For example:

With the overloads of the Export method you can export the document to an HTML string or a file,